contracts/lib/zeppelin/token/MintableToken.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.4.18;
```

Comment:
Rather than concating `Ownable` and `MintableToken`, could we split these two out? I assume you just copy pasta'd these contracts from zeppelin?

Also, maybe it'd be useful if we put these somewhere else—e.g. `contracts/staging` or etc to signal that they're only meant for test networks?

Reply:
which other contracts would fall under `contracts/staging`?

I put it inside the same file on purpose because I didn't want aOS to have an `Ownable.sol` ;)

Reply:
Offline discussion: we're moving this to aragon-apps instead.